app/views/unattended/provisioning_templates/snippet/_kickstart_networking_setup.erb

```diff
@@ -37,8 +37,9 @@ kind: snippet
 <%-   next if !interface.managed? || (interface.subnet.nil? && interface.subnet6.nil?) -%>
 <%-   next if bonded_interfaces.include?(interface.identifier) -%>
 
+<%-   interface_identifier = @host.bond_interfaces.map { |i| i.identifier }.include?(interface.attached_to) ? interface.identifier : nil %>
```

Comment:
wouldn't it be simpler (and potentially faster) to check `
@host.bond_interfaces.any?{|i| i.identifier == interface.attached_to}`?

Reply:
I guess this is not the place for review comments, as this is just a sync from community templates, where the proper reviews should take place

Reply:
+1 to what @inecas says, also `any?` is not allowed in safemode and you code is not equivalent, since we need the identifier, it could have been written used `find`/`detect` but that's also not allowed, I'd not be too concerned about optimization on this level, there are small number of interfaces